tests/appsec/waf/test_telemetry.py

```diff
@@ -1,20 +1,20 @@
 from utils import interfaces, released, rfc, weblog, scenario, context
-from utils.tools import logger
 
 TELEMETRY_REQUEST_TYPE_GENERATE_METRICS = "generate-metrics"
+TELEMETRY_REQUEST_TYPE_DISTRIBUTIONS = "distributions"
 
 
-def _validate_headers(headers):
+def _validate_headers(headers, request_type):
     """https://github.com/DataDog/instrumentation-telemetry-api-docs/blob/main/GeneratedDocumentation/ApiDocs/v2/how-to-use.md
     """
     # empty value means we don't care about the content, but we want to check the key exists
     expected_headers = {
         "Content-type": "application/json",
-        "DD-Telemetry-API-Version": "v2",
-        "DD-Telemetry-Request-Type": TELEMETRY_REQUEST_TYPE_GENERATE_METRICS,
+        "DD-Telemetry-API-Version": "v1",
```

Comment:
At some point we might have a mixture of tracers supporting v2 and some of them not yet migrated and still in v1, will it be a problem from the system-tests point of view?

Reply:
No it doesn't, good point @manuel-alvarez-alvarez! I explain it in a CAVEAT:
```
# CAVEAT: each library could have different metrics result. If you get an error ping in slack
```
The code is ready to add conditions with `if context.library == "xxx"....` if it is needed for each library and different use cases